server/routes/viewModels/taskLists/caTasks.js

```diff
@@ -17,6 +17,35 @@ const createLicence = require('./tasks/createLicence')
 const finalChecks = require('./tasks/finalChecks')
 
 module.exports = {
+  getCaTasksEligibilityLduInactive: ({ optedOut, eligible, decisions, tasks }) => {
+    const { eligibility, optOut } = tasks
+    const eligibilityDone = eligibility === 'DONE'
+    const optOutUnstarted = optOut === 'UNSTARTED'
```

Comment:
there's no need for any of these booleans, we can see this as being a static view. 
We always want all these tasks to be visible.

Reply:
I'm not so sure about this one. 
IWhen doing this section I followed the example of the getCaTasksEligibility function - line 49 - and it shows the visibility is not static because it too checks the eligibility and optOut

Reply:
oh maybe we need to use that eligibility == done to prevent locking people out before they have finished the eligibility section? so rather than just checking is stage == eligible we need to check this as well (the bug we were talking about yesterday) - don't need to deal with this in this PR though.

Either way - the idea is that we only show this list of tasks when the user cannot do any additional tasks. So the list of task we display to the user should be static? we might be able to make this simpler than `getCaTasksEligiblity`. I might have got this wrong though.

Reply:
done as per ...
1. pass in any errors to to getTaskListModel() in server/routes/taskList.js
getTaskListModel() returns a fixed array if errors are passed in. The fixed array is similar to the one returned by  caTasks.getCaTasksEligibilityLduInactive() previously
2. removed the caTasks.getCaTasksEligibilityLduInactive() function completely as no longer needed